src/main/java/io/github/fileanalysissuite/adaptersdk/interfaces/extensibility/FileMetadata.java

```diff
@@ -86,7 +85,7 @@
      *
      * @return the additional metadata
      */
-    default Map<String, Serializable> getAdditionalMetadata()
+    default Map<String, Iterable<String>> getAdditionalMetadata()
```

Comment:
I know I approved this but in retrospect, now that I've looked at the `adaptersdk-impl-jaxrs` changes, I think it would be better to change it to:

```suggestion
    default Map<String, ? extends Iterable<String>> getAdditionalMetadata()
```

Reply:
Ah, of course, I forgot about that construct